index.html

```diff
@@ -0,0 +1,177 @@
+            <div class="left-block">
+                <p class="Tonic">Multi-Post <br> Stories</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-3"></div>
+            <div class="left-block">
+                <p class="Tonic">Tonic</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+        <div class="card-work-2">
+            <div class="work-image-4"></div>
+            <div class="left-block">
+                <p class="Tonic">Multi-Post <br> Stories</p>
+                <div class="frame1">
+                    <p class="canopy">CANOPY</p>
+                    <p class="back-end-dev">Back End Dev</p>
+                    <p class="year">2015</p>
+                </div>
+                <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="tags">
+                    <a class="html">html</a>
+                    <a class="css">css</a>
+                    <a class="javascript">javascript</a>
+                </div>
+                <button class="button">See Project</button>
+            </div>
+        </div>
+    </section>
+    <section>
+        <div class="about-myself">
+            <div class="myself">
+                <p class="about-title">About <br> Myself</p>
+                <p class="des">Hello I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+            </div>
+            <div class="Right contact">
+                <a class="media-text">LET’S CONNECT</a>
```

Comment:
- [ ] Please note that this is not a link, its a title for the social links below it, so instead of wrapping it in a `<a>` tag it should a `<hx>` tag, kindly fix it